homeassistant/components/onkyo/manifest.json

```diff
@@ -7,5 +7,19 @@
   "integration_type": "device",
   "iot_class": "local_push",
   "loggers": ["pyeiscp"],
-  "requirements": ["pyeiscp==0.0.7"]
+  "requirements": ["pyeiscp==0.0.7"],
+  "ssdp": [
+    {
+      "manufacturer": "ONKYO",
```

Comment:
This does not cover other used manufacturer names, like:
- Onkyo & Pioneer Corporation
- Pioneer

There may also be some others.

Reply:
Ah good call, do you have any recommendations where I could find a list of other manufacturers? This was the only one I have access to.